modules/model/src/test/scala/io/janstenpickle/trace4cats/model/ArbitraryAttributeValues.scala

```diff
@@ -0,0 +1,52 @@
+package io.janstenpickle.trace4cats.model
```

Comment:
All changes due to `kittens` are above, all due to `scalacheck-shapeless` below.